CONTRIBUTING.md

```diff
@@ -1,20 +1,20 @@
 # Contributing Guide
+
+If you want to help develop this project, and you don't know where to start, consider:
+- checking issues with the label [`good first issue`](https://github.com/software-mansion/protostar/labels/good%20first%20issue)
+- increasing test coverage
+- fixing small bugs or adding small features
 ## Documentation fixes
 If you spot a typo, broken link, a mistake in the documentation, feel free to [create a PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
 ## New features or fixes
 Check if your issue is listed on the [Issues](https://github.com/software-mansion/protostar/issues) page. If the issue exists and is assigned to somebody, it will be resolved in the near future. Otherwise:
 1. Create a new issue.
-1. Read [the development guide](https://github.com/software-mansion/protostar#development)
-1. Use `development` as a base branch.
+1. Read [the development guide](https://github.com/software-mansion/protostar#development).
```

Comment:
1, 2, 3, 4 ...

Reply:
I think markdown converter will automatically replace it with correct numeration.

Reply:
Okey